specification/Commented_Example.nef

```diff
@@ -95,15 +95,15 @@ read seq protein.seq
       # Data history loop
```

Comment:
why don't we move this nef_run_history to the end of the file? This will favour human readability. The reason is it has a tag for the script. If am not wrong, the user may post 100s of lines of  script as a value for the tag. As a human you open the file and you see a thousand line script and the actual data you are interested is just a few lines at the bottom of the file. So think about it, its just a suggestion.